### ISSUE TITLE
new DiffEditor component doesn't accept `onChange` prop

### ISSUE THREAD

**Issue by epferrari** [securingsincity/react-ace#427]
# Problem

With the recently introduced `DiffEditor` component, there doesn't appear to be a mechanism to propagate a change in the editor out of the component. The `onChange` handler passed to the inner `SplitEditor` only sets the `DiffEditor`'s state.

https://github.com/securingsincity/react-ace/blob/0a8001fc26d179dcbbe201bf2edc1a86b65f4260/src/diff.js#L16-20

## Sample code

```typescript
public render(): React.ReactElement<any> {
  return <DiffEditor value={['hello', 'hello world']} onChange={this.handleChange}/>;
}

private handleChange(leftValue: string, rightValue: string): void {
  /* I'm never invoked :( */
}
```

**Comment by VijayKrish93** [comment]
This issue shall be closed once #428 is merged . Thanks @epferrari !

**Comment by securingsincity** [comment]
Thanks @VijayKrish93 . this is live in the latest version!
